docs/about/learning-resources.rst

```diff
@@ -7,7 +7,6 @@ Learning Resources
    :hidden:
 
    /topics
-   /videos/index
```

Comment:
This is still used in production, just not built locally. 

https://github.com/writethedocs/www/blob/14230d289fba7c5889b2d2bf252c269efa2c4477/docs/conf.py#L97-L104